src/mixins/common-mixin.ts

```diff
@@ -87,6 +88,7 @@ export class CommonMixin extends Vue {
       case CorpTypeCd.BC_ULC_COMPANY: return EntityTypes.UL
       case CorpTypeCd.COOP: return EntityTypes.CP
       case CorpTypeCd.EXTRA_PRO_A: return EntityTypes.XUL
+      case CorpTypeCd.PARTNERSHIP: return EntityTypes.GP
```

Comment:
I found, while testing, that I could not change the name of FM1017603 (a GP). This missing code is why.

At the same time, I added the reverse mapping on line 66.

Reply:
I looked at all possible BcMapping and XproMapping entity types and added the missing ones above.

Reply:
(There are 25 cases in the switch above and 20 cases in the switch here. That's because the first one has 5 "same as".)